VideoAPI/VideoApi.UnitTests/Controllers/Message/RemoveMessagesForConferenceTests.cs

```diff
@@ -0,0 +1,57 @@
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Net;
+using Task = System.Threading.Tasks.Task;
+
+using VideoApi.DAL.Queries;
+using VideoApi.Domain;
+using VideoApi.DAL.Commands;
+
+namespace VideoApi.UnitTests.Controllers.Message
+{
+    public class RemoveMessagesForConferenceTests : MessageControllerTestBase
+    {
+        [Test]
+        public async Task Should_return_badrequest_with_invalid_conference_id()
+        {
```

Comment:
The integration API tests already do this. It's just a duplicate test but with mocks

Reply:
Removed the test